### ISSUE TITLE
phylosift search ; align workflow broken

### ISSUE THREAD

**Issue by koadman** [gjospin/PhyloSift#440]
It looks like `Utilities::previous_run_handler()` is not aware of the workflow used to align isolate genomes:

```
phylosift search --isolate --besthit B_thetaiotaomicron.fna
phylosift align --isolate --besthit B_thetaiotaomicron.fna

A previous run was found using the same output name. Aborting the current run.
Either delete that run from /home/koadman/data/kfed/groopm/bin_1_phylogeny/PS_temp/B_thetaiotaomicron.fna/alignDir, or force overwrite with the -f command-line option
 at /home/koadman/git/PhyloSift/bin/../lib/Phylosift/Phylosift.pm line 308
```

**Comment by koadman** [comment]
needing to use `--force` is counter-intuitive for this workflow, even if documented. I think the previous run check only needs to be applied in `phylosift all` mode. People using `search` and `align` steps individually should know enough about what they're doing that they don't need warnings about overwriting files.

**Comment by gjospin** [comment]
Fixed in bc2dfd2a2fd08484cfa714a42828dd1bbf014139

Added the creation of sub directories only when needed (during all mode or each individual step).
Removed the -f requirement when running individual steps after search.
